doc/UsersGuide/source/CodeOverview.rst

```diff
@@ -36,6 +34,18 @@ The ufs-weather-model repository supports the short- and medium-range UFS applic
     - https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere
   * - Stochastic physics pattern generator
     - https://github.com/noaa-psd/stochastic_physics
+  * - Modular Ocean Model (MOM6)
+    - https://github.com/NOAA-EMC/MOM6
+  * - Los Alamos sea ice model (CICE6)
+    - https://github.com/NOAA-EMC/CICE
+  * - EMC WaveWatch III Model (WW3)
```

Comment:
it should be all upper case WAVEWATCH III (WAVE-height, WATer depth and Current Hindcasting)

Reply:
I will make this change. I also re-organized the ww3 inputs section a bit so please confirm that the changes make sense.

Reply:
Should it be NWS WAVEWATCH III ? or NOAA WAVEWATCH III ?

Reply:
NOAA/NCEP WAVEWATCH III (WAVE-height, WATer depth and Current Hindcasting)

Reply:
> I will make this change. I also re-organized the ww3 inputs section a bit so please confirm that the changes make sense.

It makes sense.